### PR TITLE
[zh-cn]Update extensible-admission-controllers.md

### PR DESCRIPTION
**Pull request by suning0**

### CODE REVIEW

**Comment by tengqm** [comment]
Why are we updating the `content/zh/...` directory rather than the `content/zh-cn/...` directory?

**Comment by suning0** [comment]
> Why are we updating the `content/zh/...` directory rather than the `content/zh-cn/...` directory?

Sry, I did not rebase the upstream main.

Done already.

File: content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 1]
```diff
@@ -861,10 +861,10 @@ Use the object selector only if the webhook is opt-in, because end users may ski
 跳过准入 Webhook。
 
 <!--
-This example shows a mutating webhook that would match a `CREATE` of any resource with the label `foo: bar`:
+This example shows a mutating webhook that would match a `CREATE` of any resource (but not subresources) with the label `foo: bar`:
 -->
-这个例子展示了一个 mutating webhook，它将匹配带有标签 `foo:bar` 的任何资源的
-`CREATE` 的操作：
+这个例子展示了一个 mutating webhook，它将匹配带有标签 `foo:bar` 的任何资源（但不包括子资源）的
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md:866]:
```suggestion
这个例子展示了一个变更性质的 Webhook，它将匹配带有标签 `foo:bar` 的所有资源（但不包括子资源）的
```

**Pull request merged**